octane/evmengine/types/types.go

```diff
@@ -1,6 +1,17 @@
 package types
 
-import "github.com/ethereum/go-ethereum/common"
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type WithdrawalKeeper interface {
```

Comment:
provide impls, receive interfaces.

Rather define the interface in the package that receives it, ie. bankwrap.